Makefile

```diff
@@ -1,5 +1,5 @@
 CI_BUILD_NUMBER ?= $(USER)-snapshot
-VERSION ?= 1.5.$(CI_BUILD_NUMBER)
```

Comment:
This should be a major version bump, right?

Reply:
You think this deserves a 2.0? cc: @mmcgahan

Reply:
yeah, since it's a breaking change for consumers that use inline loaders, good make that clear with semver